controls/bsi_app_4_4.yml

```diff
@@ -217,9 +217,12 @@ controls:
         • Logging of changes
         • Regular data backups.
     notes: >-
-      TBD
-    status: pending
-    rules: []
+      This requirement needs to be adressed in the respective separate systems.
+      However, one requirement can be checked automated: Encrypted communication all image registries.
+    status: automated

```

Comment:
following https://github.com/ComplianceAsCode/content/blob/master/docs/manual/developer/03_creating_content.md#reporting-status I would see the status as `partial`

Reply:
Indeed! Thanks for pointing out